bin/build-plugin-zip.sh

```diff
@@ -113,8 +113,7 @@ build_files=$(
 	ls build/*/*.{js,css,asset.php} \
 	build/block-library/blocks/*.php \
 	build/block-library/blocks/*/block.json \
-	build/block-library/blocks/*/*.css \
-	build/block-library/blocks/*/*.js \
+	build/block-library/blocks/*/*.{js,css,asset.php} \
 	build/edit-widgets/blocks/*/block.json \
 	build/widgets/blocks/*.php \
```

Comment:
Think we need to change this line too.

Reply:
It warns when files are not present:

<img width="1333" alt="Screen Shot 2021-09-17 at 10 10 10" src="https://user-images.githubusercontent.com/699132/133748261-e74bd384-f601-4d82-8852-7cbb3cc52e0d.png">

I think this is why the list for `edit-widgets` and `widgets` is shorter.

I would appreciate if we could use the following:

```shell
build_files=$(
	ls build/*/*.{js,css,asset.php} \
	build/{block-library,edit-widgets,widgets}/blocks/*.php \
	build/{block-library,edit-widgets,widgets}/blocks/*/block.json \
	build/{block-library,edit-widgets,widgets}/blocks/*/*.{js,css,asset.php} \
)
```

Unless there is a nice way to hide those warnings.